jiant/scripts/download_data/datasets/nlp_tasks.py

```diff
@@ -0,0 +1,129 @@
+"""Use this for tasks that can be obtained from NLP without further/special processing"""
+
+import jiant.scripts.download_data.utils as download_utils
+import jiant.utils.python.io as py_io
+
+
+NLP_CONVERSION_DICT = {
+    # === GLUE === #
+    "cola": {
+        "path": "glue",
+        "name": "cola",
+        "field_map": {"sentence": "text"},
+        "label_map": {0: "0", 1: "1"},
+    },
+    "mnli": {
+        "path": "glue",
+        "name": "mnli",
+        "label_map": {0: "contradiction", 1: "entailment", 2: "neutral"},
```

Comment:
Looks like `label_map` must match the task ID_TO_LABEL. Seems like `label_map` should be changed to `MnliTask.ID_TO_LABEL`.

Reply:
Good point. (Should I be working off your commits now, or do you want to make these changes?)

Reply:
I'll try pushing to the PR branch (will likely need your approval).